core/services/ocrcommon/data_source.go

```diff
@@ -33,37 +33,46 @@ type inMemoryDataSource struct {
 	monitoringEndpoint commontypes.MonitoringEndpoint
 }
 
-// dataSource uses inMemoryDataSource and implements capturing the result to be stored in the DB
-type dataSource struct {
+type dataSourceBase struct {
 	inMemoryDataSource
 	runResults chan<- pipeline.Run
 }
 
-// dataSourceV2 implements dataSource with the proper Observe return type
+// dataSourceV2 implements dataSourceBase with the proper Observe return type for ocr1
```

Comment:
typo in comment dataSourceV2 -> dataSource

Reply:
Fixed in [`7581ff2` (#8959)](https://github.com/smartcontractkit/chainlink/pull/8959/commits/7581ff2e8c10c735c1560553aa79ce3cf1b9def4)